table/tables/tables.go

```diff
@@ -54,6 +59,9 @@ type Table struct {
 	indexPrefix     kv.Key
 	alloc           autoid.Allocator
 	meta            *model.TableInfo
+
+	// partitionExpr caches partition expression.
+	partitionExpr []expression.Expression
```

Comment:
s/partitionExpr/partitionDefExprs/ ?

Reply:
```
CREATE TABLE test.t2 (id int(11)) PARTITION BY RANGE ( id ) (
		PARTITION p0 VALUES LESS THAN (6),
		PARTITION p3 VALUES LESS THAN MAXVALUE
)
```

partitionDefExprs is `id`,  while partitionExpr is 
`id < 6`
`id < MAXVALUE`

Reply:
partitionLocateExprs?

Reply:
But it is a very specific name, and `partitionExpr` is so confusing.

Reply:
But it is a very specific name, and `partitionExpr` is so confusing.